sample/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 
     defaultConfig {
         applicationId "com.klinker.android.send_message.sample"
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
Note that any app which consumes this change will have to bump the minSdkVersion to 21 (Android 5.0).